ci-osl/README.md

```diff
@@ -21,116 +21,107 @@ Warning: this image does *not* contain OpenShadingLanguage itself as it is used
 Contains:
 * boost-1.61.0
 * cmake-3.9.4
-* cppunit-1.14.0
-* glfw-3.1.2
 * python-2.7.15
-* tbb-2017_U6
+* qt-5.6.1
 * openexr-2.2.1
-* blosc-1.5.0
+* oiio-2.0.8
+* partio-1.10.1
 * numpy-1.14
 * vfxplatform-2018
 
 ## [aswf/ci-osl:2019-clang6.3](https://hub.docker.com/r/aswf/ci-osl/tags?page=1&name=2019-clang6.3)
 Contains:
 * boost-1.66.0
 * cmake-3.12.4
-* cppunit-1.14.0
-* glfw-3.1.2
 * python-2.7.15
-* tbb-2018
+* qt-5.12.6
 * openexr-2.3.0
-* blosc-1.5.0
+* oiio-2.0.8
+* partio-1.10.1
 * numpy-1.14
 * vfxplatform-2019
 
 ## [aswf/ci-osl:2019-clang7.3](https://hub.docker.com/r/aswf/ci-osl/tags?page=1&name=2019-clang7.3)
 Contains:
 * boost-1.66.0
 * cmake-3.12.4
-* cppunit-1.14.0
-* glfw-3.1.2
 * python-2.7.15
-* tbb-2018
+* qt-5.12.6
 * openexr-2.3.0
-* blosc-1.5.0
+* oiio-2.0.8
+* partio-1.10.1
 * numpy-1.14
 * vfxplatform-2019
 
 ## [aswf/ci-osl:2019-clang8.3](https://hub.docker.com/r/aswf/ci-osl/tags?page=1&name=2019-clang8.3)
 Contains:
 * boost-1.66.0
 * cmake-3.12.4
-* cppunit-1.14.0
-* glfw-3.1.2
 * python-2.7.15
-* tbb-2018
+* qt-5.12.6
 * openexr-2.3.0
-* blosc-1.5.0
+* oiio-2.0.8
+* partio-1.10.1
 * numpy-1.14
 * vfxplatform-2019
 
 ## [aswf/ci-osl:2019-clang9.3](https://hub.docker.com/r/aswf/ci-osl/tags?page=1&name=2019-clang9.3)
 Contains:
 * boost-1.66.0
 * cmake-3.12.4
-* cppunit-1.14.0
-* glfw-3.1.2
 * python-2.7.15
-* tbb-2018
+* qt-5.12.6
 * openexr-2.3.0
-* blosc-1.5.0
+* oiio-2.0.8
+* partio-1.10.1
 * numpy-1.14
 * vfxplatform-2019
 
 ## [aswf/ci-osl:2019-clang10.3](https://hub.docker.com/r/aswf/ci-osl/tags?page=1&name=2019-clang10.3)
 Contains:
 * boost-1.66.0
 * cmake-3.12.4
-* cppunit-1.14.0
-* glfw-3.1.2
 * python-2.7.15
-* tbb-2018
+* qt-5.12.6
 * openexr-2.3.0
-* blosc-1.5.0
+* oiio-2.0.8
+* partio-1.10.1
 * numpy-1.14
 * vfxplatform-2019
 
 ## [aswf/ci-osl:2020-clang7.3](https://hub.docker.com/r/aswf/ci-osl/tags?page=1&name=2020-clang7.3)
 Contains:
 * boost-1.70.0
 * cmake-3.18.4
-* cppunit-1.15.1
-* glfw-3.1.2
 * python-3.7.3
-* tbb-2019_U6
+* qt-5.12.6
 * openexr-2.4.0
-* blosc-1.5.0
+* oiio-2.1.13.0
+* partio-1.10.1
 * numpy-1.16
 * vfxplatform-2020
 
 ## [aswf/ci-osl:2021-clang10.3](https://hub.docker.com/r/aswf/ci-osl/tags?page=1&name=2021-clang10.3)
 Contains:
 * boost-1.70.0
 * cmake-3.18.4
-* cppunit-1.15.1
-* glfw-3.1.2
 * python-3.7.3
-* tbb-2019_U6
+* qt-5.12.8
 * openexr-2.4.0
-* blosc-1.5.0
+* oiio-2.1.13.0
```

Comment:
Don't hold this up, but next time you're visiting, can we bump the *2021* so that it's using an OIIO in a recent 2.2 flavor?